### ISSUE TITLE
How to test my trained model on my data set

### ISSUE THREAD

**Issue by PL-96** [pytorch/vision#1384]

**Comment by pmeier** [comment]
You need to provide more information so we can help you. Do you simply want the accuracy on the test set?

- If you are a beginner, have a lookat [this official tutorial](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#) and in particular at [this section](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#test-the-network-on-the-test-data)
- If you are familiar with CNNs and just want to know how to test your model, have look at this [reference script](https://github.com/pytorch/vision/blob/b9cbc227eed76c4f300ace18d481d03e50a4d56e/references/classification/train.py#L50-L69), which is used to train the models provided by `torchvision`

**Comment by fmassa** [comment]
Closing following the comments from @pmeier .
